model/host/host.go

```diff
@@ -37,7 +37,9 @@ type Host struct {
 	Tag             string        `bson:"tag" json:"tag"`
 	Distro          distro.Distro `bson:"distro" json:"distro"`
 	Provider        string        `bson:"host_type" json:"host_type"`
-	IP              string        `bson:"ip_address" json:"ip_address"`
+	// holds ipv6 address
```

Comment:
can you be more specific, like "ip holds the ipv6 address when applicable"?

Reply:
done